pkg/model/hornet/solid_entry_points.go

```diff
@@ -36,6 +36,11 @@ func (s *SolidEntryPoints) Contains(txHash Hash) bool {
 	return exists
 }
 
+func (s *SolidEntryPoints) Index(txHash Hash) (bool, milestone.Index) {
+	index, exists := s.entryPointsMap[string(txHash)]
+	return exists, index
```

Comment:
why swap? go convention seems to be value, bool

Reply:
We have somewhere the `GetConfirmed()` which is also `bool, index`

Reply:
yes, but those are 2 values we return there, not a "what you requested does not exist" boolean flag